### DISCUSSION TITLE
Have you considered mentioning this on Quake One?

### DISCUSSION THREAD

**Discussion by SecularSteve** [MissLav/LibreQuake#148] [General]
https://quakeone.com/

I've searched and scoured the place, and I can't find any mentions of it.

It's still fairly active, and it seems kind of logical to have a thread on there too about this amazing little endeavour.

Personally I'm more surprised that it hasn't been tried before any of this, a lot earlier at that, than that it hasn't been mentioned already.

I mean, it could bring more people over here to help out. Seeing as what Bethesda's going through recently, I doubt "id" will have a say in anything if they start reigning in their properties at their behest (no pun intended).

But that's just me. I don't want it to be anyone's choice but yours.

**Comment by lavenderdotpet** [comment]
not on quakeone yet but we have made posts slipseer and moddb now 

https://www.slipseer.com/index.php?resources/librequake.426/

https://www.moddb.com/mods/librequake/news/librequake-queer-sh

**Comment by lavenderdotpet** [comment]
ill think about creating a page on the now that ive made a few others

**Comment by lavenderdotpet** [comment]
closed (not planned)
